blob_io_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2022 Yasuhiro Matsumoto <mattn.jp@gmail.com>.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file.
+
+//go:build cgo
+// +build cgo
+
+package sqlite3
+
+import (
+	"bytes"
+	"database/sql"
+	"io"
+	"testing"
+)
+
+func TestBlobIO(t *testing.T) {
+	driverName := "sqlite3_TestBlobIO2"
+	var driverConn *SQLiteConn
+	sql.Register(driverName, &SQLiteDriver{
+		ConnectHook: func(conn *SQLiteConn) error {
+			driverConn = conn
```

Comment:
Please use [`Conn.Raw`](https://pkg.go.dev/database/sql#Conn.Raw) instead, as that is also what we should be suggesting actual clients use.